kinds/playlist/playlist_kind.cue

```diff
@@ -1,8 +1,11 @@
 package kind
 
-name:        "Playlist"
-maturity:    "merged"
-description: "A playlist is a series of dashboards that is automatically rotated in the browser, on a configurable interval."
+name:              "Playlist"
+maturity:          "merged"
+description:       "A playlist is a series of dashboards that is automatically rotated in the browser, on a configurable interval."
+pluralName:        "Playlists"
+machineName:       "playlist"
+pluralMachineName: "playlists"
```

Comment:
What's the value of explicitly writing these in? All of these are the enforced (`machineName` and `pluralMachineName`) or default (`pluralName`) values, as specified by kindsys